src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -492,15 +527,20 @@ def fuzz(self,
            extra_env=None):
     """LibFuzzerCommon.fuzz override."""
     self._test_ssh()
+    self._init_corpus_info(corpus_directories)
+    self._push_corpora_from_host_to_target()
 
-    #TODO(flowerhack): Pass libfuzzer args (additional_args) here
-    return_code = self.fuzzer.start(additional_args)
+    # Run the fuzzer.
+    return_code = self.fuzzer.start(
+        ['data/' + self.new_corpus_relative_dir_target] + additional_args)
```

Comment:
What about the other corpus dirs? If they're still somehow taken into account.. this is still very confusing. Can we make this take in all the corpus dirs explicitly?

Reply:
the new code now passes in all of the corpus directories rather than just a single one